pages/components/cards.js

```diff
@@ -228,6 +261,32 @@ const Card = styled.div`
   list-style-type: none;
 `;
 
```

Comment:
you are using a lot of div elements in this components. Try to work with semantic HTML instead.
When you display text, for instance, a p-tag is a lot better